packages/components/tests/integration/components/hds/table/th-sort-test.js

```diff
@@ -8,37 +8,46 @@ module('Integration | Component | hds/table/th-sort', function (hooks) {
   // since we'd have to repeat this for each test, we'll set it up once to be used in all tests
   hooks.beforeEach(function () {
     this.set('setSortBy', function (sortBy) {
-      return sortBy || 'artist';
+      return sortBy?.noop();
```

Comment:
This was not the suggested change. 

Let's discuss in person tomorrow, I'll set up a meeting.

Reply:
I'll file an issue to make the change, but since that change is about the component and this PR is about the tests, let's keep it separate. Resolving this for now.